server/engine/src/main/java/org/eclipse/lsp/cobol/service/settings/SettingsService.java

```diff
@@ -49,4 +49,14 @@ public interface SettingsService {
    */
   CompletableFuture<List<Object>> fetchConfigurations(List<String> sections);
 
+  /**
+   * Fetch the required configuration sections from the client.The {@link SettingsParametersEnum#LSP_PREFIX LSP prefix}
+   * will be added to each specified section.
+   *
+   * @param scopeUri the scope URI.
+   * @param sections the required sections.
+   * @return a list of configuration objects.
+   */
+  CompletableFuture<List<Object>> fetchConfigurations(String scopeUri, List<String> sections);
```

Comment:
Actually, it is a document URI. A little bit confusing that here it is called "scopeUri"